### PR TITLE
Add `examples/openssh.rs`

### LINKED ISSUE #93: exaxmples/ folder demonstrating how to use this crate.
There are tests here but reading them is just leaving me more confused: https://github.com/openssh-rust/openssh-sftp-client/blob/main/tests/highlevel.rs
It seems like we are spawning a server but I thought this was a client library? 

I think a simple example demonstrating how to use this library in the most common use case would go a long way.

### PR DESCRIPTION
**Pull request by NobodyXu**
Fixed #93

### CODE REVIEW

**Comment by NobodyXu** [comment]
@rukai I added a simple example using the `openssh` feature, which is the most simple way to construct a `Sftp`.

**Comment by NobodyXu** [comment]
@rukai You could use `Sftp::new`, but then you would have to manually spawns sftp subsystem on remote, so I just provide a simple one for starter.

**Comment by silver-ymz** [comment]
It uses some extra libraries and features. Maybe it's more friendly to use a whole project including `Cargo.toml`. Since that, users can directly run it.

**Comment by NobodyXu** [comment]
> It uses some extra libraries and features.

I added comment at top, specifying how to configure `Cargo.toml` dependencies

> Maybe it's more friendly to use a whole project including `Cargo.toml`. Since that, users can directly run it.

Hmmm, I'm not sure on this since users usually expect examples at `examples/*.rs`.

File: examples/openssh.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+//! Expected `Cargo.toml` dependencies configuration:
+//!
+//! ```
+//! [dependencies]
+//! openssh-sftp-client = { version = "0.13.6", features = ["openssh"] }
+//! ```
+
+#[cfg(not(feature = "openssh"))]
+compile_error!("Please enable features openssh!");
```
Comment (silver-ymz) [inline examples/openssh.rs:9]: This requires users to enable the openssh feature for their own crate, not for `openssh-sftp-client`. I think it's better to remove it.

File: examples/openssh.rs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+//! Expected `Cargo.toml` dependencies configuration:
+//!
+//! ```
+//! [dependencies]
+//! openssh-sftp-client = { version = "0.13.6", features = ["openssh"] }
```
Comment (silver-ymz) [inline examples/openssh.rs:5]: Maybe we should also add `tokio` and `openssh`.

File: examples/openssh.rs (hunk 1 above)
Comment (NobodyXu) [reply examples/openssh.rs]: Thanks, I've removed it.

File: examples/openssh.rs (hunk 2 above)
Comment (NobodyXu) [reply examples/openssh.rs:5]: Thanks, I've added it.

**Review by silver-ymz — APPROVED** [review]
Great

**Pull request merged**